src/services/requestService.js

```diff
@@ -39,8 +38,38 @@ export default class RequestService {
     return updatedRequest[1][0].dataValues;
   }
 
+  /**
+  * update trip request
+  * @param {number} id - request id
+  * @param {object} data - request object
+  * @return {object} - updated request
+  */
+  static async respondToRequest({ body: { status }, params: { requestId }, user: { id } }) {
+    const userRequest = await Request.findOne({ where: { id: requestId } });
+
+    if (!userRequest) error('Trip request not found');
+
+    if (status !== 'approved' && status !== 'rejected') error('Response status must be approved or rejected');
```

Comment:
i think you can validate this input with our validation middleware. you can have a schema like this  
`status: Joi.string().trim().valid('approved', 'rejected').lowercase().required()`

Reply:
Alright